lib/msal-browser/docs/create-pca.md

```diff
@@ -0,0 +1,34 @@
+# Creating Client Application instances for MSAL JS
```

Comment:
There's already an[ initialization doc ](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md)- can we update that with these changes?

Reply:
Done.